### PR TITLE
Cookie notice

### PR DESCRIPTION
**Pull request by monkemedia**

### CODE REVIEW

File: tests/features/pages/home-page.feature [hunk 1]
```diff
@@ -10,3 +10,13 @@ Feature: Home page
     When I open `/en`
     Then I see a `browse page`
     And I am on an accessible page
+  
+  Scenario: Seeing the Cookie Disclaimer on first visit
+    The homepage is a browse page from Contentful
+
+    When I open `/en`
+    And I see a `cookie disclaimer`
+    And I click `cookie disclaimer button`
```
Comment (lbiedinger) [inline tests/features/pages/home-page.feature]: ...click _the_ ...

File: components/generic/CookieDisclaimer.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+    mounted() {
+      this.init();
+    },
+
+    methods: {
+      init() {
+        const status = this.getCookieStatus();
+
+        if (!status) {
```
Comment (lbiedinger) [inline components/generic/CookieDisclaimer.vue]: is it worth doing an explicit check here for status !== 'accepted' ?

File: components/generic/CookieDisclaimer.vue [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        return !this.isOpen ? `bottom: -${this.bannerHeight}px` : '';
+      }
+    },
+
+    mounted() {
+      this.init();
+    },
+
+    methods: {
+      init() {
```
Comment (rwd) [inline components/generic/CookieDisclaimer.vue]: The statements in this method may as well just go into `mounted`.

File: components/generic/CookieDisclaimer.vue [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    name: 'CookieDisclaimer',
+
+    components: {
+      SmartLink
+    },
+
+    data() {
+      return {
+        isOpen: false,
+        COOKIE_CONSENT: 'cookie_consent',
```
Comment (rwd) [inline components/generic/CookieDisclaimer.vue]: For consistency with other stored data, change this to 'cookieConsent'.

**Pull request merged**